LabController/src/bkr/labcontroller/provision.py

```diff
@@ -92,132 +100,157 @@ def clear_orphaned_commands(self):
             self.mark_command_aborted(id, "Command orphaned, aborting")
 
     def poll(self):
-        logger.debug('Clearing orphaned commands')
+        logger.debug("Clearing orphaned commands")
         self.clear_orphaned_commands()
 
-        logger.debug('Polling for queued commands')
+        logger.debug("Polling for queued commands")
         for command in self.get_queued_commands():
-            if command['id'] in self.commands:
+            if command["id"] in self.commands:
                 # We've already seen it, ignore
                 continue
             # This command has to wait for any other existing commands against the
             # same system, to prevent collisions
-            predecessors = [self.greenlets[c['id']]
-                    for c in six.itervalues(self.commands)
-                    if c['fqdn'] == command['fqdn']]
-            if 'power' in command and command['power'].get('address'):
+            predecessors = [
+                self.greenlets[c["id"]]
+                for c in six.itervalues(self.commands)
+                if c["fqdn"] == command["fqdn"]
+            ]
+            if "power" in command and command["power"].get("address"):
                 # Also wait for other commands running against the same power address
-                predecessors.extend(self.greenlets[c['id']]
-                        for c in six.itervalues(self.commands)
-                        if 'power' in c and c['power'].get('address')
-                            == command['power']['address'])
+                predecessors.extend(
+                    self.greenlets[c["id"]]
+                    for c in six.itervalues(self.commands)
+                    if "power" in c
```

Comment:
I would have expected an indent here on if.  However, this is the original code.